sbin/usg

```diff
@@ -309,10 +311,12 @@ while true ; do
             output_flag="$1 $2"; shift 2 ;;
         --tailoring-file)
             tailoring_file=$2; shift 2 ;;
-	--html-file)
-	    HTML_FILE=$2; shift 2 ;;
-	--results-file)
-	    RESULTS_FILE=$2; shift 2;;
+        --html-file)
+	        HTML_FILE=$2; shift 2 ;;
+        --results-file)
+            RESULTS_FILE=$2; shift 2;;
```

Comment:
This change is tabs -> spaces; the other lines in this section were already space indents.